dcap/quoteverify/src/lib.rs

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) 2022 MobileCoin Foundation
+
+#![doc = include_str!("../README.md")]
+#![deny(missing_docs, missing_debug_implementations)]
+
+use mc_sgx_dcap_quoteverify_types::Path as QvPath;
+use mc_sgx_dcap_types::Quote3Error;
+use mc_sgx_util::ResultInto;
+use std::{ffi::CString, os::unix::ffi::OsStrExt, path::Path};
+
+/// Set path for QVE(Quote verification enclave) or QPL(Quote provider library)
+///
+/// This allows one to override the path that will be searched for each
+/// `path_type`.  When this isn't called then the local path and dlopen search
+/// path will be utilized.
+///
+/// Returns [`Quote3Error`] when
+/// * `path` does not point to a file
+/// * `path` is longer than 259 (bytes)
+/// * `path` contains a null (0) byte.
+///
+/// # Arguments
+/// * `path_type` - Which path to set
+/// * `path` - The path value to use.  This is the full path to a file.
+pub fn set_path<P: AsRef<Path>>(path_type: QvPath, path: P) -> Result<(), Quote3Error> {
```

Comment:
Hm. Is there any Rust idiomatic guidance on global configuration methods like this? It feels a bit sketchy as it's modifying global state, but doing it secretly. I am wondering if we should make some kind of Qv struct to attach this method and the follow-on `load_policy` and `supplemental_data_size` to. That would allow us to mark it as `!Send` (actually I think that happens automatically) and force people to think before pulling the rug out from under another thread.

Reply:
created #145